calico_containers/calico_ctl/node.py

```diff
@@ -75,6 +76,11 @@
 KUBERNETES_PLUGIN_DIR = '/usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/'
 KUBERNETES_PLUGIN_DIR_BACKUP = '/etc/kubelet-plugins/calico/'
 
+RKT_PLUGIN_VERSION = 'v0.1.0'
+RKT_BINARY_URL = 'https://github.com/projectcalico/calico-rkt/releases/download/%s/calico_rkt' % RKT_PLUGIN_VERSION
+RKT_PLUGIN_DIR = '/usr/lib/rkt/plugins/net/'
+RKT_CONF_DIR = '/etc/rkt/net.d/'
```

Comment:
I left this in for the possibility of making a default/template conf file as we discussed yesterday